### PR TITLE
Make compilation on FreeBSD work

### PR DESCRIPTION
**Pull request by huntears**
That is WIP but I hope I can merge that :)
Normally nothing should break, but I will test on linux before anything else

### CODE REVIEW

**Comment by huntears** [comment]
After some testing looks like this doesn't break linux support.

**Review by STMiki — APPROVED** [review]
Lol

**Comment by Trompettesib** [comment]
image
👀

**Review by Trompettesib — APPROVED** [review]
LGTM

File: cubic-server/interface/LogsInterface.cpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ bool LogsInterface::on_timeout(){
     while (std::getline(file_stream, logs)) {
```
Comment (NekoSelena) [inline cubic-server/interface/LogsInterface.cpp:32]: You didn't updated your branch with master; this function is not up to date (l. 30 nt supposed to be here anymore)

**Review by NekoSelena — CHANGES_REQUESTED** [review]
You need to pull master; the logs interface is not up to date

**Comment by huntears** [comment]
> image eyes

yeah i commited on my FreeBSD VM, which doesn't have my gpg key yet xd

File: cubic-server/interface/LogsInterface.cpp (hunk 1 above)
Comment (huntears) [reply cubic-server/interface/LogsInterface.cpp:32]: Let me do that now :)

**Review by STMiki — APPROVED** [review]
Still lol

**Review by Trompettesib — APPROVED** [review]
LGTM

**Review by NekoSelena — APPROVED** [review]

**Pull request merged**